src/dodal/beamlines/p38.py

```diff
@@ -1,12 +1,49 @@
-from dodal.devices.areadetector import AdAravisDetector
+from pathlib import Path
+
+from ophyd_async.core import DeviceCollector
+from ophyd_async.epics.areadetector import (
+    ADDriver,
+    HDFStreamerDet,
+    NDFileHDF,
+    TmpDirectoryProvider,
+)
+from ophyd_async.panda import PandA
+
+from dodal.devices.athena.panda import FlyingPanda
 from dodal.utils import BeamlinePrefix
 
-PREFIX: str = BeamlinePrefix("p38").beamline_prefix
+BEAMLINE = "p38"
+PREFIX: str = BeamlinePrefix(f"{BEAMLINE}").beamline_prefix
+VISIT = "2023/cm33874-3"  # TODO will need to be derived from visit service
```

Comment:
remove the TODO as it's no longer relevant

Reply:
Done